app/src/main/java/org/p2p/wallet/settings/ui/recovery/RecoveryKitPresenter.kt

```diff
@@ -0,0 +1,25 @@
+package org.p2p.wallet.settings.ui.recovery
+
+import android.os.Build
+import org.p2p.wallet.auth.repository.UserSignUpDetailsStorage
+import org.p2p.wallet.common.mvp.BasePresenter
+
+class RecoveryKitPresenter(
+    private val signUpDetailsStorage: UserSignUpDetailsStorage,
+) : BasePresenter<RecoveryKitContract.View>(),
+    RecoveryKitContract.Presenter {
+
+    override fun attach(view: RecoveryKitContract.View) {
+        super.attach(view)
+        signUpDetailsStorage.getLastSignUpUserDetails()?.let { userDetails ->
+            with(view) {
+                showDeviceName(
+                    Build.MANUFACTURER + " " + Build.MODEL
+                )
+                // TODO PWN-5216 show phone and device from request!
```

Comment:
We don't have a request now? 
If not, just hide it. 
We are in prod, we shouldn't show stub data

Reply:
fixed